src/components/RequestReview/RequestReview.tsx

```diff
@@ -36,7 +36,7 @@ export const RequestReview: React.FunctionComponent<IRequestReviewProps> = (prop
                 setRequest(newRequest);
                 setUserCanReview(checkIfUserCanReview(newRequest));
             } else {
-                history.push("/Requests");
+                redirect("/Requests");
```

Comment:
Since this is when an item is _not_ found, should this be a history.replace("/Requests"); ? (Back button will not navigate to this "bad" item)